wagtailio/blog/models.py

```diff
@@ -1,22 +1,41 @@
 from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
 from django.db import models
 from django.shortcuts import render
-
-from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
+from django.utils.functional import cached_property
+
+from modelcluster.fields import ParentalKey
+from wagtail.admin.edit_handlers import (
+    FieldPanel,
+    InlinePanel,
+    PageChooserPanel,
+    StreamFieldPanel,
+)
 from wagtail.core.fields import StreamField
-from wagtail.core.models import Page
+from wagtail.core.models import Orderable, Page
 from wagtail.images.edit_handlers import ImageChooserPanel
 from wagtail.snippets.edit_handlers import SnippetChooserPanel
 from wagtail.snippets.models import register_snippet
 
 from wagtail_airtable.mixins import AirtableMixin
 from wagtail_content_import.models import ContentImportMixin
 
-from wagtailio.utils.blocks import StoryBlock
+from wagtailio.blog.blocks import BlogStoryBlock
 from wagtailio.utils.mappers import StreamFieldMapper
 from wagtailio.utils.models import CrossPageMixin, SocialMediaMixin
 
 
+class BlogPageCategory(models.Model):
```

Comment:
Is the plan to potentially allow more than one category? if so, it could use a note.
Otherwise a simple `ForeignKey` to `taxonomy.Category` would do

Reply:
Thanks, updated in 127f2be to use a `ForeignKey` to `taxonomy.Category`